.github/workflows/check-projects.yml

```diff
@@ -45,6 +45,7 @@ jobs:
           - javascript-vanilla-with-webpack
           - typescript-angular
           - typescript-lit-element
+          - typescript-moduleResolution-bundler
```

Comment:
Until we have a released version of bpmn-visualization including the fix, we should exclude to build this project with the release version.
And restore it when the new release is available.